### DISCUSSION TITLE
EDC Presentation

### DISCUSSION THREAD

**Discussion by lucian-torje-siemens** [eclipse-edc/.github#27] [General]
Hello @jimmarino @juliapampus @ndr-brt ,
I'm working on a presentation about EDC for my colleagues - in case you have something similar that you can share please help me out.

**Comment by juliapampus** [comment]
Please find all slides in our [collateral repo](https://github.com/eclipse-dataspaceconnector/Collateral) (`events` and `latest presentations`).

**Reply by lucian-torje-siemens** [reply]
Thank you @juliapampus
